pom.xml

```diff
@@ -13,7 +13,8 @@
 
     <groupId>io.jenkins.plugin-modernizer</groupId>
     <artifactId>plugin-modernizer-parent-pom</artifactId>
-    <version>${revision}${changelist}</version>
```

Comment:
This is the version show when the jar is executed with -V option

Reply:
Maven inherit the version.

Confirmed filtering works with 

```
java -jar plugin-modernizer-cli/target/jenkins-plugin-modernizer-6.ve1a_9d7f37c6f.jar --version
6.ve1a_9d7f37c6f
```